src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlSerializationWriterILGen.cs

```diff
@@ -2425,6 +2486,7 @@ internal void ILGenForCreateInstance(CodeGenerator ilg, Type type, Type cast, bo
             ilg.Call(Activator_CreateInstance);
             if (cast != null)
                 ilg.ConvertValue(Activator_CreateInstance.ReturnType, cast);
+            ilg.MarkLabel(labelReturn);
```

Comment:
I've verified a few more scenarios and verify that the find/use code seem to always covers Activator.CreateInstance. I think the CreateInstance part can be removed.